### PR TITLE
Fix display issues

### PR DESCRIPTION
**Pull request by Kiwi-x-Kiwi**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import GameDetails from './pages/GameDetails'
 
 import { library } from '@fortawesome/fontawesome-svg-core'
 import { fab } from '@fortawesome/free-brands-svg-icons'
-import { faBars, faHeart, faGripHorizontal, faGripLines } from '@fortawesome/free-solid-svg-icons'
+import { faBars, faHeart, faGripHorizontal, faGripLines} from '@fortawesome/free-solid-svg-icons'
```
Comment (bobbypwang) [inline client/src/components/App.jsx]: Please add the space back.

File: client/src/components/Games/GamesList.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .pagination{
     display: inline-block;
-    margin-right: 5px;
+    // margin-right: 5px;
```
Comment (bobbypwang) [inline client/src/components/Games/GamesList.scss]: Don't worry. Be happy, just delete it. We have git for a reason.

File: client/src/components/generic/CollapsibleList.jsx [hunk 3]
```diff
@@ -1,13 +1,15 @@
 import React, { useState, useRef } from "react";
 import './CollapsibleList.scss'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCaretDown, faCaretUp  } from '@fortawesome/free-solid-svg-icons'
```
Comment (bobbypwang) [inline client/src/components/generic/CollapsibleList.jsx]: Add this to App.jsx so you don't have to call it on each page that might need it.

**Review by bobbypwang — CHANGES_REQUESTED** [review]
Minor changes to the structure. Everything else looks good.

File: client/src/components/App.jsx (hunk 1 above)
Comment (Kiwi-x-Kiwi) [reply client/src/components/App.jsx]: Sorry!  Added back.

File: client/src/components/generic/CollapsibleList.jsx (hunk 3 above)
Comment (Kiwi-x-Kiwi) [reply client/src/components/generic/CollapsibleList.jsx]: Awesome, didn't know about that.  Added to App.jsx.  Thanks!

File: client/src/components/Games/GamesList.scss (hunk 2 above)
Comment (Kiwi-x-Kiwi) [reply client/src/components/Games/GamesList.scss]: Deleted 👍

**Pull request merged**